script/Ciel/Day 15/15_01B_CIEL15B_1.txt

```diff
@@ -3090,6 +3101,7 @@ I'll have to do something about that as well.
 [sha:23b76677f96eed5d1f6361ac226654fbd14a0316]{
 -- Page 188, Offset 42521.
 -- 「すみません。いま言葉を濁したところ、はっきり口にしてくれますか？」
+// Make this a bit more humorous?
```

Comment:
Don't love this but
> Sorry, could you say that again? I'm a little worried that I heard you right the first time.

Reply:
I was actually talking about the below comment.
I think this one works just fine.

42522
>I didn't mishear her mention some kind of vampire-specific disciplining, did I?